client/screens/Dashboard.jsx

```diff
@@ -16,7 +16,6 @@ export function Dashboard() {
 
   return (
     <Container component="main" maxWidth="xs">
-      <CssBaseline />
```

Comment:
I removed this from some of the files as a test and didn't notice a change at all. Not sure what it does.

Reply:
Looking at the docs, as long as we have one of them somewhere it's sufficient for the [CSS reset/baseline](https://material-ui.com/components/css-baseline/) to set standard styles